src/status.h

```diff
@@ -41,6 +41,8 @@ struct status_main {
 	int compat;
 	int pass;
 	int progress;
+	int resume_salt;
+	uint32_t *resume_salt_md5;
```

Comment:
It was put there because of this (from recovery.c)

```C
	fprintf(rec_file, "%u\n%u\n%x\n%x\n%x\n%x\n%x\n%x\n%x\n"
	    "%d\n%d\n%d\n%x\n",
	    status_get_time() + 1,
	    status.guess_count,
	    status.combs.lo,
	    status.combs.hi,
	    status.combs_ehi,
	    status.crypts.lo,
	    status.crypts.hi,
	    status.cands.lo,
	    status.cands.hi,
	    status.compat,
	    status.pass,
	    status_get_progress ? (int)status_get_progress() : -1,
	    rec_check);
```

Everything else about the status of the current run's state is stored there.  The only thing that is questionable is the resume_salt flag (the trigger to cracker.c).  I placed that flag here out of convenience.  The pointer certainly belongs here.